ras_party/models/models.py

```diff
@@ -322,6 +322,22 @@ def to_respondent_dict(self):
             "password_reset_counter": self.password_reset_counter,
         }
 
+    def to_dict(self) -> dict:
```

Comment:
This clearly isn't ideal, as we have a to_respondent_dict as well. The problem with that one is not only assign party_id to id, but also the keys are a mix of camelCase and snake, it also returns an odd UNIT_TYPE which is a constant and therefore not persisted in the db. Naturally I can't just change to_respondent_dict as it will break anything calling to it.

However as this new endpoint is a migration piece, eventually to_respondent_dict can be removed.